### DISCUSSION TITLE
Martin Adminer crashes clean install

### DISCUSSION THREAD

**Discussion by rnrkrft** [wintercms/winter#852] [Q&A] [answered]
I tried to install Martin.Adminer, as I saw it in [TheFehr](https://github.com/TheFehr) list of plugins he uses. But it I ran into an error: 

![Bildschirmfoto 2023-03-01 um 17 36 33](https://user-images.githubusercontent.com/6118278/222207540-955dd414-148a-48cf-a883-dac4f81f62ca.png)

**Comment by LukeTowers** [comment, marked as answer]
Submit an issue to [skydiver/october-plugin-adminer](https://github.com/skydiver/october-plugin-adminer). All they have to do to fix it is remove that __construct() method entirely, it's not even meant for that plugin, it looks copy/pasted from their SSOLogin plugin.

**Reply by rnrkrft** [reply]
Sorry was [TheFehr](https://github.com/TheFehr).

**Reply by LukeTowers** [reply]
Submit an issue to https://github.com/skydiver/october-plugin-adminer. All they have to do to fix it is remove that __construct() method entirely, it's not even meant for that plugin, it looks copy/pasted from their SSOLogin plugin.

**Reply by rnrkrft** [reply]
Thank you for the help. It installs now... But it does not reach the login and ends in a blank screen

**Reply by LukeTowers** [reply]
Do you have debug mode enabled? I can't help you with that little of information to go off of.
